blog/_posts/2017-03-14-scala-native-0.1-is-here.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+post-type: blog
+by: Denys Shabalin
+title: "Scala Native v0.1 is here!"
+---
+
+We're excited to announce the first release of
+[Scala Native](http://www.scala-native.org), an optimising ahead-of-time
+compiler for Scala built on top of the [LLVM](http://llvm.org) compiler
+infrastracture.
+
+The project was originally announced during Scala Days New York 2016 and
+now has reached a point of relative feature completeness.
```

Comment:
Added some background information in https://github.com/scala/scala-lang/pull/618/commits/d1c8928e4108fd4508cb13e564e3799beb50e06d